<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera List : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCameraList">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>15</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>16</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>21</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PixelFormat :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPixelFormat">
   <property name="geometry">
    <rect>
     <x>584</x>
     <y>16</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>17</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>17</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="laPic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>47</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>836</x>
     <y>77</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[  Gain ( 0 ~ 511)  ]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editGain">
   <property name="geometry">
    <rect>
     <x>853</x>
     <y>107</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkGainAuto">
   <property name="geometry">
    <rect>
     <x>1009</x>
     <y>109</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGain">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>107</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkShutterAuto">
   <property name="geometry">
    <rect>
     <x>1009</x>
     <y>189</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>836</x>
     <y>157</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[  Shutter ( 1 ~ 3600000000)  ]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShutter">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>187</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editShutter">
   <property name="geometry">
    <rect>
     <x>853</x>
     <y>187</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>836</x>
     <y>240</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[  Trigger  ]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>302</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Source : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkTriggerEnable">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>333</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameter : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>370</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fire S/W Trigger :</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>857</x>
     <y>400</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRunSW">
   <property name="geometry">
    <rect>
     <x>982</x>
     <y>364</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run S/W Trigger</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTriggerSource">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>299</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editTriggerParameter">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTriggerPara">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>330</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="laRecvCount">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>440</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Recieve Frame Count : </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
